src/test/rpc/Transaction_test.cpp

```diff
@@ -692,6 +693,64 @@ class Transaction_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testRequest(FeatureBitset features)
+    {
+        testcase("Test Request");
+
+        using namespace test::jtx;
+        using std::to_string;
+
+        const char* COMMAND = jss::tx.c_str();
+
+        Env env{*this};
+        Account const alice{"alice"};
+        Account const alie{"alie"};
+        Account const gw{"gw"};
+        auto const USD{gw["USD"]};
+
+        env.fund(XRP(1000000), alice, gw);
+        env.close();
+
+        // AccountSet
+        env(noop(alice));
+
+        // Payment
+        env(pay(alice, gw, XRP(100)));
+
+        std::shared_ptr<STTx const> txn = env.tx();
+        env.close();
+        std::shared_ptr<STObject const> meta =
+            env.closed()->txRead(env.tx()->getTransactionID()).second;
+
+        Json::Value expected = txn->getJson(JsonOptions::none);
+        expected[jss::DeliverMax] = expected[jss::Amount];
+
+        auto const result =
+            env.rpc(COMMAND, to_string(txn->getTransactionID()));
+        BEAST_EXPECT(result[jss::result][jss::status] == jss::success);
+
+        for (auto memberIt = expected.begin(); memberIt != expected.end();
+             memberIt++)
+        {
+            std::string const name = memberIt.memberName();
+            if (BEAST_EXPECT(result[jss::result].isMember(name)))
+            {
+                auto const received = result[jss::result][name];
+                std::ostringstream ssReceived;
+                ssReceived << received;
+                std::ostringstream ssExpected;
+                ssExpected << *memberIt;
```

Comment:
You can do away with the `operator<<` with `Json::Value`'s `to_string` overload. It reduces the number of lines. If you like it, here's the [change](https://github.com/ckeshava/rippled/tree/json2Str)

Reply:
Oh cool - did not know that!